### ISSUE TITLE
[Feature Request] Adding missing voices in home page

### ISSUE THREAD

**Issue by mikk64** [michaelherger/lms-plugin-tidal#91]
in my personal home page, in original Tidal android app, i have few "voices" i can't find in the plugin "Home Page" link:
- "Album che adorerai" (something like "albums you'll love", differents from "new albums for you")
- "Appena aggiornato" (something like "just updated", referring to playlists created by Tidal )
there are others links, but these are the ones i'd like to know if there is any chance to have in the plugin.
many thanks

**Comment by michaelherger** [comment]
I tried to find what you refer to (the title's you're referring to are Italian, aren't they?). But I don't see them. Could you please provide a screenshot of what you're seeing?

TIDAL (like many other music services) use some non-documented calls to provide the home menu. We have to figure out from the data files we're receiving what a menu item should or could be. Therefore we'll never be on par with what they provide.

**Comment by mikk64** [comment]
thanks, Michael, to look at this
yes, that voices are from italian android version
i look at it, this morning, and they are not in the windows app version, on my pc
these are the screenshots, taken from android app:

![Screenshot (28 ago 2024 08_32_29)](https://github.com/user-attachments/assets/b91e7dc6-8042-4f49-a6ac-35add47bd27b)
![Screenshot (28 ago 2024 08_33_19)](https://github.com/user-attachments/assets/277ca859-47a1-4cba-8ed3-a4541a83fb6a)

not sure if they are visible, i'm just seeing letters and numbers...
anyway, i see that the links in the lms plugin are exactly the same of my windows app, and they differ from the ones on android
thanks anyway, you are precious!

**Comment by michaelherger** [comment]
Would the same suggestions be on the web UI, but under a different label? Unless I can find them in the web UI it'll be close to impossible to even investigate them.

**Comment by mikk64** [comment]
no, they're not, just in android app, the web UI has the same voices of the window app

as you suggest, i looked around and i've found out, in the link Explore, or search, i'm not sure in english, inside there is "Suggerimenti" and there, i can see, among others, "Album storici", that is targeted on my tastes:
![1](https://github.com/user-attachments/assets/26177bf6-7f51-4652-8959-163dceb481fb)
![2](https://github.com/user-attachments/assets/a06b9aac-e247-41b0-9cc8-4df2d756167a)

can you see it too, if you use tidal, of course?
it's interesting, but of course it's another thing from the one i meant
